### PR TITLE
Small docs improvements

### PR DESCRIPTION
**Pull request by JanuszL**
- fix path in the examples to the files in the repository
- improve `last_batch_padded` description for MXNet plugin
- small wording correction

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It fixes a small bugs in docs

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
       fix path in the examples to the files in the repository
       improve `last_batch_padded` description for MXNet plugin
       small wording correction
 - Affected modules and functionalities:
     MXNet plugin
     examples
 - Key points relevant for the review:
     NA
 - Validation and testing:
     CI
 - Documentation (including examples):
     This change affects only documentation

**JIRA TASK**: *[NA]*

### CODE REVIEW

**Comment by JanuszL** [comment]
!build

File: dali/python/nvidia/dali/plugin/mxnet.py [hunk 1]
```diff
@@ -214,10 +214,11 @@ class DALIGenericIterator(_DALIIteratorBase):
...
                  data from the current epoch is dropping padding samples or samples from
-                 the next epoch. If set to False next epoch will end sooner as data from
-                 it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 the next epoch (it doesn't literally drop but set `pad` field of ndarray
+                 so the latter code could use it to drop the data). If set to False next
+                 epoch will end sooner as data from it was consumed but dropped. If set to
+                 True next epoch would be the same length as the first one. For this happen,
+                 the option `pad_last_batch` in the reader needs to be set to `True` as well.
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/mxnet.py:221]:
```suggestion
                 the next epoch (it doesn't literally drop, but sets the ``pad`` field of ndarray,
                 so the following code could use it to drop the data). If set to ``False`` next
                 epoch will end sooner, as data from it was consumed but dropped. If set to
                 ``True``, next epoch would be the same length as the first one. For this to happen,
                 the option ``pad_last_batch`` in the reader needs to be set to ``True`` as well.
```

File: dali/python/nvidia/dali/plugin/paddle.py [hunk 2]
```diff
@@ -178,7 +178,7 @@ class DALIGenericIterator(object):
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
                  same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 in the reader needs to be set to `True` as well.
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/paddle.py:181]:
```suggestion
                 in the reader needs to be set to ``True`` as well.
```

File: dali/python/nvidia/dali/plugin/paddle.py [hunk 3]
```diff
@@ -446,7 +446,8 @@ class DALIClassificationIterator(DALIGenericIterator):
                  data from the current epoch is dropping padding samples or samples from
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one.
+                 same length as the first one. For this happen, the option `pad_last_batch`
+                 in the reader needs to be set to `True` as well.
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/paddle.py:450]:
```suggestion
                 same length as the first one. For this happen, the option ``pad_last_batch``
                 in the reader needs to be set to ``True`` as well.
```

File: dali/python/nvidia/dali/plugin/pytorch.py [hunk 4]
```diff
@@ -112,7 +112,7 @@ class DALIGenericIterator(object):
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
                  same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 in the reader needs to be set to `True` as well.
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/pytorch.py:115]:
```suggestion
                 in the reader needs to be set to ``True`` as well.
```

File: dali/python/nvidia/dali/plugin/pytorch.py [hunk 5]
```diff
@@ -345,7 +345,8 @@ class DALIClassificationIterator(DALIGenericIterator):
                  data from the current epoch is dropping padding samples or samples from
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one.
+                 same length as the first one. For this happen, the option `pad_last_batch`
+                 in the reader needs to be set to `True` as well.
```
Comment (szalpal) [inline dali/python/nvidia/dali/plugin/pytorch.py:349]:
```suggestion
                 same length as the first one. For this happen, the option ``pad_last_batch``
                 in the reader needs to be set to ``True`` as well.
```

File: dali/python/nvidia/dali/plugin/mxnet.py (hunk 1 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/mxnet.py]: Done

File: dali/python/nvidia/dali/plugin/paddle.py (hunk 2 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/paddle.py]: Done

File: dali/python/nvidia/dali/plugin/paddle.py (hunk 3 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/paddle.py]: Done

File: dali/python/nvidia/dali/plugin/pytorch.py (hunk 4 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/pytorch.py]: Done

File: dali/python/nvidia/dali/plugin/pytorch.py (hunk 5 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/pytorch.py]: Done

File: dali/python/nvidia/dali/plugin/mxnet.py [hunk 6]
```diff
@@ -211,13 +211,14 @@ class DALIGenericIterator(_DALIIteratorBase):
...
                  if the iterator returning last batch with data only partially filled with
                  data from the current epoch is dropping padding samples or samples from
-                 the next epoch. If set to False next epoch will end sooner as data from
-                 it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 the next epoch (it doesn't literally drop but sets ``pad`` field of ndarray
+                 so the following code could use it to drop the data). If set to ``False`` next
+                 epoch will end sooner as data from it was consumed but dropped. If set to
+                 ``True`` next epoch would be the same length as the first one. For this happen,
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/mxnet.py:220]:
```suggestion
                 ``True`` next epoch would be the same length as the first one. For this to happen,
```

File: dali/python/nvidia/dali/plugin/mxnet.py [hunk 7]
```diff
@@ -450,12 +451,14 @@ class DALIClassificationIterator(DALIGenericIterator):
...
+                 or it just wraps up. In the conjunction with ``fill_last_batch`` it tells
                  if the iterator returning last batch with data only partially filled with
                  data from the current epoch is dropping padding samples or samples from
-                 the next epoch. If set to False next epoch will end sooner as data from
-                 it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one.
+                 the next epoch (it doesn't literally drop but sets ``pad`` field of ndarray
+                 so the following code could use it to drop the data). If set to ``False`` next
+                 epoch will end sooner as data from it was consumed but dropped. If set to
+                 ``True`` next epoch would be the same length as the first one. For this happen,
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/mxnet.py:460]:
```suggestion
                 ``True`` next epoch would be the same length as the first one. For this to happen,
```

File: dali/python/nvidia/dali/plugin/mxnet.py [hunk 8]
```diff
@@ -564,13 +567,14 @@ class DALIGluonIterator(_DALIIteratorBase):
...
                  if the iterator returning last batch with data only partially filled with
                  data from the current epoch is dropping padding samples or samples from
-                 the next epoch. If set to False next epoch will end sooner as data from
-                 it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 the next epoch (it doesn't literally drop but sets ``pad`` field of ndarray
+                 so the following code could use it to drop the data). If set to ``False`` next
+                 epoch will end sooner as data from it was consumed but dropped. If set to
+                 ``True`` next epoch would be the same length as the first one. For this happen,
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/mxnet.py:576]:
```suggestion
                 ``True`` next epoch would be the same length as the first one. For this to happen,
```

File: dali/python/nvidia/dali/plugin/paddle.py [hunk 9]
```diff
@@ -177,8 +177,8 @@ class DALIGenericIterator(object):
                  data from the current epoch is dropping padding samples or samples from
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 same length as the first one. For this happen, the option ``pad_last_batch``
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/paddle.py:180]:
```suggestion
                 same length as the first one. For this to happen, the option ``pad_last_batch``
```

File: dali/python/nvidia/dali/plugin/paddle.py [hunk 10]
```diff
@@ -446,7 +446,8 @@ class DALIClassificationIterator(DALIGenericIterator):
                  data from the current epoch is dropping padding samples or samples from
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one.
+                 same length as the first one. For this happen, the option ``pad_last_batch``
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/paddle.py:449]:
```suggestion
                 same length as the first one. For this to happen, the option ``pad_last_batch``
```

File: dali/python/nvidia/dali/plugin/pytorch.py [hunk 11]
```diff
@@ -111,8 +111,8 @@ class DALIGenericIterator(object):
                  data from the current epoch is dropping padding samples or samples from
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one. For this happen, the option `pad_last_batch`
-                 in the reader need to be set to `True` as well.
+                 same length as the first one. For this happen, the option ``pad_last_batch``
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/pytorch.py:114]:
```suggestion
                 same length as the first one. For this to happen, the option ``pad_last_batch``
```

File: dali/python/nvidia/dali/plugin/pytorch.py [hunk 12]
```diff
@@ -345,7 +345,8 @@ class DALIClassificationIterator(DALIGenericIterator):
                  data from the current epoch is dropping padding samples or samples from
                  the next epoch. If set to False next epoch will end sooner as data from
                  it was consumed but dropped. If set to True next epoch would be the
-                 same length as the first one.
+                 same length as the first one. For this happen, the option ``pad_last_batch``
```
Comment (mzient) [inline dali/python/nvidia/dali/plugin/pytorch.py:348]:
```suggestion
                 same length as the first one. For this to happen, the option ``pad_last_batch``
```

File: dali/python/nvidia/dali/plugin/mxnet.py (hunk 6 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/mxnet.py]: Done

File: dali/python/nvidia/dali/plugin/mxnet.py (hunk 7 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/mxnet.py]: Done

File: dali/python/nvidia/dali/plugin/mxnet.py (hunk 8 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/mxnet.py]: Done

File: dali/python/nvidia/dali/plugin/paddle.py (hunk 9 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/paddle.py]: Done

File: dali/python/nvidia/dali/plugin/paddle.py (hunk 10 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/paddle.py]: Done

File: dali/python/nvidia/dali/plugin/pytorch.py (hunk 11 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/pytorch.py]: Done

File: dali/python/nvidia/dali/plugin/pytorch.py (hunk 12 above)
Comment (JanuszL) [reply dali/python/nvidia/dali/plugin/pytorch.py]: Done

**Pull request merged**